.circleci/get_docker_compose.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Usage: $ ./get-docker-compose.sh
```

Comment:
Mind if we keep the original name of this file? I'd recommend we are consistent with how bash scripts are named in this repo and continue to use hyphens (`-`) to separate words with hyphens.

Reply:
Fixed.